gitevanhsu/book-reviews
# Feature/group

**Function**
1. Book page add the group button, led member to group page.
2. Group page like chat room, members can communicate through text.
3. Also can start 1 on 1 live communication.

**Test Plan**
1. Sign in and go to group page and left some massage.
2. Use another account and go to left some massage in same group, check message update.
3. Use 1 on 1 live communication, check it work as expect.
![截圖 2022-11-08 下午2 34 13](https://user-images.githubusercontent.com/92194495/200491959-a313e3e8-e3bb-4c18-9e46-e536133b8161.png)
